forms-flow-web/src/components/FOI/customComponents/Records/util.js

```diff
@@ -1,142 +1,164 @@
 //Remove duplicate records
-export const removeDuplicateFiles = (recordList) => (
-    recordList.filter(({isduplicate}) => !isduplicate)
-  );
-  
-  // Helper function to sort files by lastmodified date
-export const sortByLastModified = (files) => (
-    files.sort((a, b) => new Date(a.lastmodified) - new Date(b.lastmodified))
-  );
+export const removeDuplicateFiles = (recordList) =>
+  recordList.filter(({ isduplicate }) => !isduplicate);
+
+// Helper function to sort files by lastmodified date
+export const sortByLastModified = (files) =>
+  files.sort((a, b) => new Date(a.lastmodified) - new Date(b.lastmodified));
 
 // Helper function to sort attachments by lastmodified date
-const sortAttachmentsByLastModified = (attachments) => (
-    attachments.sort((a, b) => new Date(a?.attributes?.lastmodified) - new Date(b?.attributes?.lastmodified))
+const sortAttachmentsByLastModified = (attachments) =>
+  attachments.sort(
+    (a, b) =>
+      new Date(a?.attributes?.lastmodified) -
+      new Date(b?.attributes?.lastmodified)
   );
-  
-export const getPDFFilePath = (item, conversionFormats) => {
- 
-    let pdffilepath = item.s3uripath
-    let pdffilename = item.filename
- 
-    if (((item.isredactionready && conversionFormats.includes(item.attributes?.extension?.toLowerCase())) ||
-    (item.attributes?.isattachment && item.attributes?.trigger === 'recordreplace'))) {
-        pdffilepath = pdffilepath.substr(0, pdffilepath.lastIndexOf(".")) + ".pdf";
-        pdffilename += ".pdf";
-    }
-    return [pdffilepath, pdffilename];
- }
-
- function arrangeAttachments(attachments, parentDocumentMasterId, conversionFormats) {
-    const attachmentsMap = {};
-    const arrangedAttachments = [];
-  
-    // Create a map of attachments based on parentid
-    for (const attachment of attachments) {
-      const parentid = attachment.parentid;
-      if (!attachmentsMap[parentid]) {
-        attachmentsMap[parentid] = [];
-      }
-      attachmentsMap[parentid].push(attachment);
+
+export const getPDFFilePath = (item) => {
+  let pdffilepath = item.s3uripath;
+  let pdffilename = item.filename;
+
+  console.log(`item = ${JSON.stringify(item)}`);
+
+  if (
+    (item.isredactionready && item.isconverted) ||
```

Comment:
removed conversionFormats.includes(item.attributes?.extension?.toLowerCase()) and replaced with item.isconverted